TRANSLATIONS.md

```diff
@@ -0,0 +1,71 @@
+# AMP Translation Initiative
+
+Welcome to the AMP Translation Initiative!
+
+AMP's mission includes making the web a welcoming and accessible place for everyone, and we are unable to obtain this goal without localized documentation. Translating AMP documentation is an essential process that aids the AMP community's success and heath.
+
+The AMP Translation Initiative is working to create a network of native speakers to ensure our documentation is localized with clarity and efficiency. We can't thank you for taking an interest in AMP and your assistance in making the web a content ecosystem for everyone.
+
+## Ways to Contribute
+
+  * Translate documentation.
+  * Update translated documentation.
+  * Review translated documentation for accuracy and terminology.
+  * Add to localization term glossaries.
+
+## Get Started
+
+AMP has an ongoing internal translation process for documentation. We are working with volunteers to form a communication line that describes what files are being translated and which ones require external assistance.
+
+### Currently, our process is:
+
+  1. Find a file to translate.
+  1. Create a [GitHub Issue](https://github.com/ampproject/docs/issues/new).
+  1. Declare which language you are translating the document.
+  1. Name which file you wish to translate or update.
+  1. Give a rough estimation of how much of the file you are translating.
+    * Full File
+    * Partial File
+    * Few Terms
+  1. Provide a general estimation of how much time it will take to complete the translation task.
+  1. Add [@CrystalFaith](https://github.com/CrystalFaith) to give document status:
```

Comment:
"..so she can triage the localization request and inform you in case there's already somebody working on the same translation."